.env.n8n.example

```
@@ -0,0 +1,41 @@
+# n8n Automation - Environment Configuration
+# Copy to .env.n8n and fill in values:
+#   cp .env.n8n.example .env.n8n
+
+# === n8n Core ===
+N8N_BASIC_AUTH_ACTIVE=true
+N8N_BASIC_AUTH_USER=forge-admin
+N8N_BASIC_AUTH_PASSWORD=CHANGE_ME_USE_STRONG_PASSWORD
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The N8N_BASIC_AUTH_PASSWORD key should go before the N8N_BASIC_AUTH_USER key

(UnorderedKey)